strawberry/channels/testing.py

```diff
@@ -109,19 +96,21 @@ async def gql_init(self) -> None:
         if self.protocol == GRAPHQL_TRANSPORT_WS_PROTOCOL:
             assert res == (True, GRAPHQL_TRANSPORT_WS_PROTOCOL)
             await self.send_json_to(
-                ConnectionInitMessage(payload=self.connection_params).as_dict()
+                transport_ws_types.ConnectionInitMessage(
+                    {"type": "connection_init", "payload": self.connection_params}
+                )
```

Comment:
I don't know if **I** prefer, but we can also do this:


```suggestion
                transport_ws_types.ConnectionInitMessage(
                    type="connection_init", payload= self.connection_params
                )
```

no? 😊